tests/suite-unit/PluginFlyvemdmAgent.php

```diff
@@ -219,4 +219,64 @@ public function testCanViewItem() {
       $testedInstance = $this->newTestedInstance;
       $this->boolean($testedInstance->canViewItem())->isFalse();
    }
+
+   public function testGetTopicsToCleanup() {
+      $topics = \PluginFlyvemdmAgent::getTopicsToCleanup();
+
+      $expected = [
+         // Commands
+         'Command/Subscribe',
+         'Command/Ping',
+         'Command/Geolocate',
+         'Command/Inventory',
+         'Command/Lock',
+         'Command/Wipe',
+         'Command/Unenroll',
+
+         // Policies
+         'Policy/passwordEnabled',
+         'Policy/passwordMinLength',
```

Comment:
Is not there a better way to manage this? I mean if the policies grow this will need it too.

Reply:
Yes and I explicitly want this. The purpose is that every topics are actually cleaned up. This is important enough to be checked IMO.